### PR TITLE
added home and login page

### PR DESCRIPTION
**Pull request by tkevans**
@runongirlrunon @JReed21 home page and login page created

### CODE REVIEW

**Review by JReed21 — CHANGES_REQUESTED** [review]
You don't need to link to the libraries over the web, we have bootstrap, jquery, and bootstraps javascript in our local folders. 

<?= $this->Html->css('bootstrap.css') ?>
<?= $this->Html->script('jquery-3.2.1.min.js') ?>


Hard coding the ctp file into href is fine for now since we have no created controllers BUT if you are working on the login page you must create a login controller to handle the authorization and login logic

**Comment by tkevans** [comment]
@JReed21 took out most href, pages still look fine

**Comment by runongirlrunon** [comment]
@jhuang28 please merge in the master branch and fix the merge conflicts.

**Comment by tkevans** [comment]
@JReed21 took out most href, pages look fine, checkboxes were replaced by radios.

**Comment by runongirlrunon** [comment]
@JReed21 could you re-review for the changes you requested?

File: src/Template/Pages/home.ctp [hunk 1]
```diff
@@ -18,114 +18,105 @@ use Cake\Core\Plugin;
...
-use Cake\Routing\Router;
 
 $this->layout = false;
 
-
+if (!Configure::read('debug')):
+    throw new NotFoundException('Please replace src/Template/Pages/home.ctp with your own version.');
+endif;
 
 $cakeDescription = 'CakePHP: the rapid development PHP framework';
```
Comment (runongirlrunon) [inline src/Template/Pages/home.ctp]: This line can be removed, as it's not used anymore.

File: src/Template/Pages/login.ctp [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+use Cake\Error\Debugger;
+use Cake\Network\Exception\NotFoundException;
+
+$this->layout = false;
+
+if (!Configure::read('debug')):
+    throw new NotFoundException('Please replace src/Template/Pages/home.ctp with your own version.');
+endif;
+
+$cakeDescription = 'CakePHP: the rapid development PHP framework';
```
Comment (runongirlrunon) [inline src/Template/Pages/login.ctp]: Same here.

**Review by runongirlrunon — CHANGES_REQUESTED** [review]

**Comment by runongirlrunon** [comment]
[Login view & controller](https://trello.com/c/hg9MQR12/9-login-view-controller)

**Review by JReed21 — DISMISSED** [review]

**Comment by tkevans** [comment]
@runongirlrunon @JReed21 homepage and login page are updated; these two pages now can connect to each other. and if register.ctp is in Pages folder then they can connect to register page as well

**Review by runongirlrunon — APPROVED** [review]

**Comment by runongirlrunon** [comment]
Note: the default template has an unnecessary green bar across the top but I'm going to ignore that for now and write a card to clean that up later.

**Pull request merged**